### ISSUE TITLE
Create "Horror Companion" Setting

### ISSUE THREAD

**Issue by Grauenwolf** [Grauenwolf/SavageTools#1]
This would be used by 

* Rippers
* Rifts
* ETU

**Comment by herrozerro** [comment]
Do you have any kind of tool or template to create a valid XML document?  I am looking to tackle this task.

**Comment by Grauenwolf** [comment]
No, sorry I don’t. Eventually I want to create a desktop version that uses the same files, but I don’t have the time right now. But you can just take your best guess, then kick it over to me to fixup.
